src/manifest.json

```diff
@@ -3,7 +3,7 @@
     "description": "The Free Software Media System",
     "lang": "en-US",
     "short_name": "Jellyfin",
-    "start_url": "index.html#!/home.html",
+    "start_url": "index.html#/home.html",
```

Comment:
Either way this should be updated, but are `#!` routes not working? We _should_ be automatically updating `#!` routes to the `#` equivalent.

Reply:
It tries to match `!/home.html` with available routes. Maybe initial navigation isn't normalized somewhere?

Reply:
Ok I'll do some additional testing on those paths but that doesn't block this. 👍